### ISSUE TITLE
Notify the user on registering organisation snaps to explain how to add collaborators

### ISSUE THREAD

**Issue by nottrobin** [canonical-ols/build.snapcraft.io#651]

**Comment by matthewpaulthomas** [comment]
So, er … How do you add collaborators? I guess it’s in the store UI somewhere. Could someone please give an example link and/or screenshot of the page? Thanks.

**Comment by evandandrea** [comment]
Hm, it's less than ideal. There's a "Collaboration" link on the developer portal. It offers a field to add new collaborators by email, which don't have to have accounts already (yay).

<img width="1440" alt="screen shot 2017-04-24 at 12 47 55" src="https://cloud.githubusercontent.com/assets/1523179/25348801/773de1a6-28ed-11e7-8144-f226a452e78a.png">

But the email they get assumes they have context for this (what's a package, what's an Ubuntu One account).

<img width="992" alt="screen shot 2017-04-24 at 12 48 11" src="https://cloud.githubusercontent.com/assets/1523179/25348818/862703c8-28ed-11e7-854a-891b707ee5aa.png">

Clicking the "accept this share" link takes them to SSO without their email address pre-filled in the form or any explanation of what's going on or what they need to do.

<img width="928" alt="screen shot 2017-04-24 at 12 48 39" src="https://cloud.githubusercontent.com/assets/1523179/25348841/916e3dc8-28ed-11e7-8d13-b20ebe9f4f4f.png">

But if they manage to get through that and do the tricky verify your email dance, it does remember to add them as a collaborator.
<img width="974" alt="screen shot 2017-04-24 at 12 56 07" src="https://cloud.githubusercontent.com/assets/1523179/25348875/b30b9ce6-28ed-11e7-8ce8-c5e80b8c3e06.png">

Which ultimately lands them on the developer portal.

<img width="961" alt="screen shot 2017-04-24 at 12 56 17" src="https://cloud.githubusercontent.com/assets/1523179/25348884/b8ac8bec-28ed-11e7-86eb-8b94649f1c39.png">

**Comment by cjwatson** [comment]
There's also [/dev/api/snaps/<snap_id>/developers](https://dashboard.snapcraft.io/docs/api/snap.html#manage-collaborators-for-a-snap-package), although I'm not sure how you're supposed to find out the other developers' account IDs, and of course they'd have to have IDs first.

**Comment by nottrobin** [comment]
Maybe it's just not feasible to explain how to add collaborators at this stage. I don't think we should be adding extensive documentation to build.snapcraft.io. Once it is effectively documented elsewhere then we can simply link there, but until it is, I suggest we just don't mention it. Maybe we leave this issue open but de-prioritise it?

**Comment by cjwatson** [comment]
There's work in progress on this from the store end: I've seen a design that would give us the ability to invite developers by email address, which looks like exactly what we need.  My wild guess is that it's probably about a week of store work, although I don't really know for sure.

Given the amount of stuff we have above this in our backlog, I suggest that we consider this blocked until that work is done, but we only need to explicitly deprioritise it if this bubbles up to the top before then.

**Comment by evandandrea** [comment]
I'm happy to mark this blocked. I don't think it's going to be the highest priority thing for us to work on for a few weeks.

@cjwatson When you say you've "seen a design that would give us the ability to invite developers by email address", do you mean as an API? Otherwise it's not clear to me how it differs to what we have now in the developer portal. Also, can you share a link? 😀

**Comment by cjwatson** [comment]
Yes, it's an API design document.  https://docs.google.com/document/d/1WWfGAjl3IWOxk9oxsn7R0q2tShQhFvCcnQe-vQ41OnM (Canonical-only)

**Comment by evandandrea** [comment]
Thank you.
